entities/service.go

```diff
@@ -36,6 +42,20 @@ func NewService(p *Pod) *v1.Service {
 	}
 }
 
+// NewService returns the service boilerplate based on service template
+func NewServiceFromTemplate(t Service) *v1.Service {
+	serviceID++
```

Comment:
these global variables will bite us back when we start to run things concurrently

Reply:
generate random uuid instead?

Reply:
will put lock on it

Reply:
done